src/sentry/event_manager.py

```diff
@@ -2127,29 +2127,30 @@ def _process_existing_aggregate(
 
 
 def _get_severity_score(event: Event) -> float | None:
-    # If the event is info-level or lower, auto-mark as low severity
-    if LOG_LEVELS_MAP[event.data["level"]] <= logging.INFO:
-        return 0
+    # Short circuit the severity value if we know the event is fatal or info/debug
+    level = event.data.get("level")
+    if LOG_LEVELS_MAP[level] == logging.FATAL:
+        return 1.0
+    if LOG_LEVELS_MAP[level] <= logging.INFO:
```

Comment:
just confirming, this includes debug?

Reply:
it'd include debug and sample

Reply:
![Screenshot 2023-11-30 at 3 54 20 PM](https://github.com/getsentry/sentry/assets/30991498/30b52e6c-abe4-4487-96c9-ace514147fb9)
yes